compiler/il/ILOpCodeProperties.hpp

```diff
@@ -1627,7 +1627,7 @@
    /* .properties4          = */ 0,
    /* .dataType             = */ TR::Int32,
    /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer,
-   /* .childProperties      = */ TWO_SAME_CHILD(TR::Int32),
+   /* .childProperties      = */ TWO_CHILD(TR::Int32, TR::Int32),
```

Comment:
I'm afraid this nothing more than an automated refactoring artifact :sweat_smile:. Fixed!